core/src/main/java/org/elasticsearch/search/aggregations/AggregationBuilder.java

```diff
@@ -69,6 +70,16 @@ public String getName() {
     /** Add a sub aggregation to this builder. */
     public abstract AggregationBuilder subAggregation(PipelineAggregationBuilder aggregation);
 
+    /** Return the configured set of subaggregations **/
+    public List<AggregationBuilder> getSubAggregations() {
```

Comment:
I'm generally on the fence about adding getters for stuff that *might* be used but isn't at the moment. It looks like this wasn't needed so far (at least there is nothing using it in this PR, might be the clients need it), so I'm +0 on adding stuff we don't really but maybe I'm missing the reason. Same for the other method here and the "getFilter()".